thesis-de/content-de.tex

```diff
@@ -1888,16 +1888,104 @@ \section{Userstories}
 Tickets arbeiten können.
 \tm%
 
-\subsection{Userstory 1}
+\subsection{SSD Userstory}
+\\textbf{Name der Story:}\\ Anzeige der SSD's, welche einen Hardwaredeffekt haben könnten.
+\\textbf{Beschreibung:}\\ In der heutigen Zeit, werden vermehrt langsame Hard Disk Drive's
+durch schnellere Solid State Disk's ausgetauscht. Gerade im Serverbetrieb, sorgt
+dies für eine schnellere Absicherung der Daten. SSDs haben jedoch, wie auch
+HDDs einen Lebenszyklus, der bei SSD schneller erreicht werden kann. Der
+Administrator soll hier darüber Benachrichtigt werden, sobald von einer SSD
+ein kritischer Schreib- oder Lesezyklus erreicht worden ist, um die SSD, bereits
+vor dem Ausfall austauschen zu können. Ebenfalls soll hier auch eine Statistik
+ausgegeben werden, welche SSD von welchem Hersteller in der Wahrscheinlichkeit
+am häufigsten ausfallen kann.
+\\textbf{Akzeptanzkriterien:}\\
 
-\subsection{Userstory 2}
+\begin{outline}
+  \1 Die Ausgabe und Visualisierung in einen Diagramm, ob Balken- oder
+  Graphendiagramm, soll jeweils für jede einzelne Instanz angezeigt werden
+  \1 Auflistung einer Statistik in Form eines Balkendiagramms, über die
+  Hersteller, welche am wahrscheinlich häufigsten in den Instanzen ausfallen
+  \1 Eine Benachrichtigung, ob per Email oder andere Kommunikationswege,
+  wenn ein kritischer Schreib- oder Lesezyklus erreicht worden ist.
+\end{outline}
+\mr%
 
-\subsection{Frontend}
-Name der Story: Visualisierung der erhaltenen Daten zur weiteren Verwendung
-Beschreibung: Dem Administrator oder dem User soll es ermöglicht werden, eine
+\subsection{CPU Userstory}
+\\textbf{Name der Story:}\\ Auswertung der CPU Auslastung von jeder Instanz, um Engpässe zu
+erkennen.
+\\textbf{Beschreibung:}\\ Die CPUs der heutigen Zeit werden immer schneller und besser, von
+der Taktrate, bis hin zur Verarbeitungsgeschwindigkeit der Prozesse. Bei 
+Virtuellen Maschine, werden die CPUs jeweils pro Virtueller Maschine zugewiesen,
+so kann eine Maschine 2 CPU Kerne mit z.B. 2,5 GHz Takrate zugewiesen werden.
+Auf jeder Virtuellen Maschine laufen andere Dienste, weshalb eine VM
+unterschiedliche CPUs zugewiesen haben. Manchmal kommt es hier vor, das eine VM
+zu viel Rechenleistung, bedingt durch ausgeführte Dienste benötigt. Wenn eine
+VM mehr Rechenleistung benötigt, als zugewiesen ist, kann es hier zu
+Verzögerungen kommen. Um dies zu erkennen, benötigt der Administrator, eine
+detailierte Übersicht in Form eines Graphen, wieviel CPU Leistung von einer
+Virtuellen Instanz verwendet wird, um hier einen Engpass bereits vorab zu
+erkennen und der VM mehr CPU Leistung zur Verfügung stellen zu können.
+\\textbf{Akzeptanzkriterien:}\\
+
+\begin{outline}
+  \1 Visualisierung der Auslastung der CPU Leistung für jede einzelne Instanz
+  \1 Alle Werte der CPU, wie soft-IRQ, wait oder interrupt müssen visualisiert
```

Comment:
ja würd ich gern aber knallt das nicht wenn ich zwei unterschiedlichen Branches in der gleichen Datei an der Gleichen stelle etwas hinzufüge? Ich habe es mir notiert und würde es gern später hinzufügen